windows/devices/uwb/UwbSession.cxx

```diff
@@ -295,3 +296,15 @@ UwbSession::DestroyImpl()
         throw e;
     }
 }
+
+std::vector<uint8_t>
+UwbSession::GetOobDataObjectImpl()
+{
+    auto applicationConfigurationParameters = GetApplicationConfigurationParameters(AllParameters);
+    auto uwbSessionData = GetUwbSessionData(applicationConfigurationParameters);
+
+    uwbSessionData.sessionId = GetId();
```

Comment:
We should figure out an appropriate default version for `uwbSessionData.sessionDataVersion`. The spec says:
> This two byte field should be interpreted as two unsigned integer numbers representing the major and minor version.

But I don't know where this version comes from. Right now, we default it to 0, which is almost certainly wrong. Their example was `0x0101` for v1.1, so I'm inclined to use that. I'll make a separate PR to do this.

Reply:
I made the change here when I fixed the Linux build.